csharp/elasticbeanstalk/elasticbeanstalk-bg-pipeline/src/ElasticbeanstalkBgPipeline/ElasticbeanstalkBgPipelineStack.cs

```diff
@@ -0,0 +1,75 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.CodePipeline.Actions;
+using Amazon.CDK.AWS.CodePipeline;
+using Amazon.CDK.AWS.CodeCommit;
+using Amazon.CDK.AWS.Lambda;
+using Amazon.CDK.AWS.S3;
+using System.Collections.Generic;
+
+namespace ElasticbeanstalkBgPipeline
+{
+    public class ElasticbeanstalkBgPipelineStack : Stack
+    {
+        public ElasticbeanstalkBgPipelineStack(Construct parent, string id, IStackProps props) : base(parent, id, props)
+        {
+            var blueEnv = this.Node.TryGetContext("blue_env");
+            var greenEnv = this.Node.TryGetContext("green_env");
+            var appName = this.Node.TryGetContext("app_name");
+
+            var bucket = new Bucket(this, "BlueGreenBucket", new BucketProps{
```

Comment:
`{` should have its own line for C#
(applies to all)

Reply:
Is there a C# style guide I can use to lint my applications. I really don't want to check out and make changes across 10 branches, just to be asked to do it again over some other C# idiom.